src/CookieThrower.cpp

```diff
@@ -0,0 +1,60 @@
+#include "CookieThrower.h"
+#include "GameManager.h"
+#include <glm/gtx/rotate_vector.hpp>
+
+/*
+ * Poll input from the given input handler, and throws a throwable
+ * dependent on the input and according constraints.
+ */
+CookieThrower::CookieThrower() {
```

Comment:
I guess intuitively it makes the most sense for the drawing of the aiming line to be associated with the "thrower"... would also allow throwers to be attached to other objects and still see the line (for ai, etc.).  Just thinking out loud, not sure if that makes sense for the overall code.